scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -103,6 +103,14 @@ abstract class ScanResultsStorage {
             properties["password"] = config.password
             properties["ApplicationName"] = "$ORT_NAME - $TOOL_NAME"
 
+            if (config.sslmode != "disabled") {
+                properties["ssl"] = "true"
+                properties["sslmode"] = config.sslmode
+                config.sslcert?.let { properties["sslcert"] = it }
+                config.sslkey?.let { properties["sslkey"] = it }
+                config.sslrootcert?.let { properties["sslrootcert"] = it }
+            }
```

Comment:
So, just to double-check, do we need an `else` branch here that sets `properties["ssl"] = "false"`?

Reply:
No, setting just this would enable SSL with the default mode "verify-all".